clients/go/cmd/zbctl/internal/commands/cancelInstance.go

```diff
@@ -11,6 +11,7 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
+
```

Comment:
@abbasadel forgot to mention it. This newline should usually not be added.

-> https://github.com/golang/go/wiki/CodeReviewComments#package-comments